### ISSUE TITLE
Compatibility with Edge Chrome

### ISSUE THREAD

**Issue by Bamajon1974** [deanoemcke/thegreatsuspender#1085]
Are there any issues of note or any specific nuances using The Great Suspender with Edge Chrome? 
Thanks in advance!

**Comment by liamjohnston** [comment]
Not sure sorry. Probably - it hasn't been developed for or tested with Edge, your mileage may vary.

**Comment by Bamajon1974** [comment]
Got it. Will this extension be tested with Edge Chrome in the near future?

Thanks, JW

**Comment by trexx** [comment]
I've been running TGS on Edge Chrome with no issues. Currently on version 80.0.361.45.
No crashes, tabs are suspended correctly and sessions are saved and restorable. Its currently my daily driver on multiple machines.

**Comment by Bamajon1974** [comment]
Great. You are running a beta version of Edge Chrome? The lack of any
issues sounds promising.

Thanks, Jon

**Comment by makedir** [comment]
how to install on edge?

**Comment by keeganwitt** [comment]
This issue and #1057 are duplicates.

**Comment by keeganwitt** [comment]
It mostly worked for me, except for internal pages, such as `edge://extensions/`.

**Comment by keeganwitt** [comment]
@makedir You can't install it on the old Edge, only the new Chromium based Edge.  If you've got that, then you install from [Chrome Store](https://chrome.google.com/webstore/category/extensions), just like Chrome.

I'm not sure if the maintainer plans on adding to [Microsoft Store](https://microsoftedge.microsoft.com/addons/category/Edge-Extensions) or not, but I'm not sure it makes any difference -- unless Microsoft's extension sync implementation (still in progress) only syncs stuff from Microsoft Store -- I haven't seen any comments on that yet.

**Comment by Bamajon1974** [comment]
The maintainer did not have any immediate plan to evaluate compatibility of
The Great Suspender with Edge Chrome but if numerous requests are made, his
priorities may shift.

Although Edge Chrome and Google Chrome are similar, MS incorporated a few
differences, most notably in terms of privacy and security and
functionality of extensions correspondingly differ.

**Comment by keeganwitt** [comment]
Fair enough. For those that find their way here from Google, there is an alternative extension that you might consider: [Auto Tab Discard](https://microsoftedge.microsoft.com/addons/detail/nfkkljlcjnkngcmdpcammanncbhkndfe).

**Comment by Bamajon1974** [comment]
How does Auto Tab Discard compare to the Great Suspender? Do the extensions
ultimately do the same thing? Is one more or less buggy or developed than
the other?

**Comment by keeganwitt** [comment]
I think both are pretty similar. I've used Auto Tab Discard for years on Firefox (my primary browser), I'd just always used The Great Suspender on Chrome (I tend to dabble in other browsers).  Both seem to work well.

The only difference I'm aware of is that selecting a tab un-hibernates it in ATD. It doesn't have the face page to click to un-suspend the tab like TGS. This has the benefit of being able to copy hibernated URLs without the extension prefix, but the disadvantage of perhaps accidentally un-suspending a tab you didn't mean to if you accidentally clicked a tab unintentionally.

Though, as I said, with the exception of internal edge pages, TGS has worked flawlessly for me in Edge so far as I've seen.  I think it'd work well enough that you don't _need_ to switch to if you don't want to.

**Comment by makedir** [comment]
Great suspender works normally on Edge and I am using it now. Edge also is way faster than Chrome because of internal Windows Hw acceleration MS implemented into it, Chrome doesnt have (so far).

**Comment by srberard** [comment]
I've been using TGS on Edge for a while and love it.  Would like to see official support for it.

It's been largely flawless until recently.  With the latest Edge Dev update, I'm getting one CPU fully pegged.  The behavior I'm seeing is similar to issue #1080 so this may not be an Edge-specific issue, but rather something related to Chromium v83.

**Comment by pilar6195** [comment]
Having high CPU usage on latest Edge Dev (83.0.467.0) as well.

One of the reviews in the chrome store from a few days ago indicate that they too are having this issue on edge.

**Comment by trexx** [comment]
One minor issue I have spotted so far is that when TGS suspends a tab viewing a page with the Immersive Reader (read:// in the address bar) it will fail to resume when you click on it.

Version: 81.0.416.45

**Comment by Bamajon1974** [comment]
Does a new request to test TGS with Edge Chrome need to be made? As more
and more folks are using Edge Chrome, I would think an evaluation for
compatibility would need to be made.

**Comment by donnyork** [comment]
The term "Edge Chrome" is mistaken (not that I'm a techie... merely a terminology stickler) and in this discussion simply means the Microsoft Edge browser -- not Google's Chrome browser.  Am I assuming this correctly? (For my part, I can't even install The Great Suspender in the Edge browser successfully.)

**Comment by srberard** [comment]
Technically, it's the new Edge browser that uses the Chromium rendering engine.  

There's instructions here detailing how to install extensions for Google Chrome in the new Edge browser:
  https://support.microsoft.com/en-us/help/4538971/microsoft-edge-add-or-remove-extensions

**Comment by donnyork** [comment]
I'm returning after my previous here, to say my Edge browser does accept the installation: I only had to copy the following from the address field displaying in Chrome: https://chrome.google.com/webstore/detail/the-great-suspender/klbibkeccnjlkjkiokjodocebajanakg?hl=en
Pasting this into the address field of a newly opened Edge browser will open a usable installation if the user executes choices just as if "Chrome" means "Edge" in the prompt or the button label.  Easy peazy!

**Comment by Bamajon1974** [comment]
Edge Chromium (or Chrome) is supposedly built on the Chrome framework so
Microsoft's implementation of the browser should function very similarly.
However, I seem to recall (cannot remember where) that there are some
differences and that Edge Chrome had different security features. So it may
be worth a request for the developer to test with Edge Chrome since there
are some differences between it and Google Chrome.
